server/tournament.py

```diff
@@ -674,12 +674,20 @@ def create_spirit_scores(spirit_score: SpiritScoreUpdateSchema) -> SpiritScore:
     )
 
     if spirit_score.mvp_id:
-        mvp = UCPerson.objects.get(id=spirit_score.mvp_id)
-        score.mvp = mvp
+        if use_uc_reg:
+            mvp = UCPerson.objects.get(id=spirit_score.mvp_id)
+            score.mvp = mvp
```

Comment:
can we also add mcp_v2 here if person - player relationship exists? So atleast from now on we can have mvp v2 as source of truth

Reply:
yeah sure